extensions/mongodb-client/runtime/src/main/java/io/quarkus/mongodb/FindOptions.java

```diff
@@ -294,9 +294,6 @@ public <T> FindPublisher<T> apply(FindPublisher<T> stream) {
         if (noCursorTimeout) {
             publisher = publisher.noCursorTimeout(true);
         }
-        if (oplogReplay) {
-            publisher = publisher.oplogReplay(true);
-        }
```

Comment:
Should we also drop the field/method in our class?

It doesn't have any replacement that we would need to support?

Reply:
Thanks for your attentiveness, yes we should drop the field. I'll take care of it.

According to the 4.11.1 Javadocs this option isn't supported on a database level from MongoDB 4.4 onwards.

No additional options were introduced between the versions.